riverdriver/river_driver_interface.go

```diff
@@ -115,11 +115,10 @@ type Executor interface {
 	JobGetByKindAndUniqueProperties(ctx context.Context, params *JobGetByKindAndUniquePropertiesParams) (*rivertype.JobRow, error)
 	JobGetByKindMany(ctx context.Context, kind []string) ([]*rivertype.JobRow, error)
 	JobGetStuck(ctx context.Context, params *JobGetStuckParams) ([]*rivertype.JobRow, error)
-	JobInsertFast(ctx context.Context, params *JobInsertFastParams) (*rivertype.JobRow, error)
-	JobInsertFastMany(ctx context.Context, params []*JobInsertFastParams) ([]*rivertype.JobRow, error)
+	JobInsertFast(ctx context.Context, params *JobInsertFastParams) (*JobInsertFastResult, error)
+	JobInsertFastMany(ctx context.Context, params []*JobInsertFastParams) ([]*JobInsertFastResult, error)
```

Comment:
This relates to the API changes from #584 so we may want to align on the right change to make here.